main.go

```diff
@@ -41,12 +41,24 @@ var defaultCollections = map[string]targetSystem{
 		notifierApp:    cmsNotifier,
 		scope:          scopeContent,
 	},
+	"spark": {
+		name:           "spark",
+		originSystemID: "cct",
+		notifierApp:    cmsNotifier,
+		scope:          scopeContent,
+	},
 	"video": {
 		name:           "video",
 		originSystemID: "next-video-editor",
 		notifierApp:    cmsNotifier,
 		scope:          scopeContent,
 	},
+	"pac-metadata": {
+		name:           "pac-metadata",
+		originSystemID: "http://cmdb.ft.com/systems/pac",
```

Comment:
cmdb.ft.com doesn't exist anymore. We can either  add the systemCode or point this field to dewey.

Reply:
@JermilaPaulDhas I think we should let this go as-is, this is only used as an identifier and the format is quite baked in to various publishing pipelines, e.g.
- http://git.svc.ft.com/projects/MSG/repos/ft-async-message/browse/src/main/java/com/ft/messaging/standards/message/v1/SystemId.java#9
- https://github.com/Financial-Times/native-ingester/blob/master/helm/native-ingester/app-configs/native-ingester-cms_publishing.yaml#L9
- https://github.com/Financial-Times/native-ingester/blob/master/helm/native-ingester/app-configs/native-ingester-metadata_publishing.yaml#L9
and possibly elsewhere.

If we want to update it, that's a _lot_ of work.